kv/tikv/config/config.go

```diff
@@ -27,7 +27,7 @@ type Config struct {
 	RaftElectionTimeoutTicks    int
 	RaftMinElectionTimeoutTicks int
 	RaftMaxElectionTimeoutTicks int
-	RaftMaxSizePerMsg           uint64
+	RaftMaxEntsSize             uint64
```

Comment:
can we remove it?

Reply:
I will remove it in a upcoming pr about `Storage` and `RaftLog`.